src/app/features/characters/components/random-character/random-character.component.ts

```diff
@@ -1,16 +1,51 @@
-import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MarvelChar } from '@characters/models/characters.model';
+import { Subscription } from 'rxjs';
+import { CharactersMarvelService } from '@characters/services/marvel.service';
 
 @Component({
   selector: 'app-random-character',
   standalone: true,
-  imports: [],
+  imports: [CommonModule],
   templateUrl: './random-character.component.html',
   styleUrl: './random-character.component.scss'
 })
-export class RandomCharacterComponent {
-  name = '';
-  description = '';
-  thumbnail = 'src';
-  homepage = 'src1';
-  wiki = 'src2';
+export class RandomCharacterComponent implements OnInit, OnDestroy {
+  id!: number;
+  subs!: Subscription;
+
+  thumbnail!: string;
```

Comment:
such kind of variables are better named with `Url` ending, like `homepageUrl` because the variable names should be descriptive

Reply:
I will move these variables later and rename them this way